extract_dmd.d

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env dub
+/++
+dub.sdl:
+dependency "libdparse" version="0.7.0-beta.7"
+dependency "mustache-d" version="0.1.3"
```

Comment:
Do I understand correctly that this adds a dependency on a 3rd-party library we don't control for the single instance of `{{ grammarTraitsList }}`? If so, a `std.array.replace` call would be more pragmatic.